### DISCUSSION TITLE
Subdivide Claim disable

### DISCUSSION THREAD

**Discussion by jekayl** [GriefPrevention/GriefPrevention#2401] [Q&A]
How do I disable subdivide claim? especially the alias. When we do /sc (message) it leads to subdivide instead.

**Comment by Jikoo** [comment]
If you're trying to override a specific alias, I have a writeup on the subject here: https://github.com/Jikoo/OpenInv/wiki/Customizing-Aliases

Otherwise, subclaims are slated to be removed with v18.
